src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -233,8 +234,8 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
   return (
     <>
       <ImageBuilderHeader inWizard />
-      <section className="pf-l-page__main-section pf-c-page__main-section">
-        <Wizard
+      <PageSection>
```

Comment:
Love this! Doesn't the `<LandingPage>` have an `isWidthLimited` prop? Would we want to put that here?